.github/workflows/docs-preview-deploy.yml

```diff
@@ -17,12 +17,10 @@ jobs:
     name: 'Deploy Preview'
     runs-on: ubuntu-22.04
     # Requires a PR event triggered `docs-preview-prepare.yml` workflow run that was successful + ensure the head SHA belongs to an associated PR:
-    if: |
-      ${{
-        github.event.workflow_run.conclusion == 'success'
-        && github.event.workflow_run.event == 'pull_request'
-        && contains(github.event.workflow_run.pull_requests.*.head.sha, github.event.workflow_run.head_sha)
-      }}
+    if: >-
+      github.event.workflow_run.conclusion == 'success'
+      && github.event.workflow_run.event == 'pull_request'
+      && contains(github.event.workflow_run.pull_requests.*.head.sha, github.event.workflow_run.head_sha)
```

Comment:
The way it was defined before it was always evaluated to true since it was a string rather than a boolean (see https://github.com/nikitastupin/pwnhub/blob/main/writings/if-condition.md)

Reply:
Ok thank you :)


```suggestion
    # Multi-line `if` GHA expression should avoid wrapping with `${{ }}`:
    # https://github.com/nikitastupin/pwnhub/blob/main/writings/if-condition.md
    if: |
      github.event.workflow_run.conclusion == 'success'
      && github.event.workflow_run.event == 'pull_request'
      && contains(github.event.workflow_run.pull_requests.*.head.sha, github.event.workflow_run.head_sha)
```

Seems like official docs don't touch on this caveat?:

- https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions
- https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/using-conditions-to-control-job-execution

Reply:
Unfortunately not